spec/mailer_spec.cr

```diff
@@ -0,0 +1,19 @@
+require "./spec_helper"
+
+describe Quartz::Mailer do
+  it "has a version" do
+    Quartz::Mailer::VERSION.should be_a String
+  end
+
+  context "deliver" do
+    it "doesnt try to connect when smtp is disabled" do
```

Comment:
@amberframework/contributors suggestions welcome for how to test this. I immediately want to reach for some sort of mock object, but I'm not sure how that is done / could be done in a compiled language.

Reply:
I don't have time at this moment but we can pair to get this working